File: clients/android/noop/src/main/kotlin/com/clienttools/sdk/ClientToolsSDK.kt
package com.clienttools.sdk

import android.content.Context
import com.clienttools.sdk.http.CustomRoute

object ClientToolsSDK {
    fun init(context: Context) {}
    fun init(context: Context, customRoutes: List<CustomRoute> = emptyList(), customHandlerTimeoutMs: Long = 4500L) {}
    fun resolveRedirect(url: String): String = url
    fun shutdown() {}
}

File: clients/ios/noop/Sources/ClientToolsSDK.swift
import Foundation

public class ClientToolsSDK {
    public static let shared = ClientToolsSDK()
    private init() {}

    public func start(port: Int = 8080, customRoutes: [Any] = [], customHandlerTimeoutMs: Int = 4500) {}
    public func resolveRedirect(_ url: String) -> String { return url }
    public func getCurrentPage() -> (pageName: String, timestamp: String) { return ("", "") }
    public func recordPageChange(_ pageName: String) {}
}
